### ISSUE TITLE
sshd example config change sshd to sshd-session

### ISSUE THREAD

**Issue by Eno77** [Aldaviva/Fail2Ban4Win#35]
since these login attempts are bugging me  i try out your solution, thanks for the effort!

But I think cyg sshd changed their source for the according log entries, see screenshot.
So you might change the example configuration accordingly.

![grafik](https://github.com/user-attachments/assets/51bf2495-23d4-4b11-b734-cc9125d4280b)

and an example event (redacted):
```
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
	<System>
		<Provider Name="sshd-session"/>
		<EventID Qualifiers="0">0</EventID>
		<Version>0</Version>
		<Level>4</Level>
		<Task>0</Task>
		<Opcode>0</Opcode>
		<Keywords>0x80000000000000</Keywords>
		<TimeCreated SystemTime="2024-08-26T12:06:54.1321465Z"/>
		<EventRecordID>3049992</EventRecordID>
		<Correlation/>
		<Execution ProcessID="0" ThreadID="0"/>
		<Channel>Application</Channel>
		<Computer>foo</Computer>
		<Security UserID="bar"/>
	</System>
	<EventData>
		<Data>sshd-session: PID 8689: Failed password for invalid user newton from *(redacted to protect the attacker) port <theport> ssh2</Data>
	</EventData>
</Event>
```

edit: be aware the data string has "sshd-session" at the beginning too, so the regex pattern may be updated too.

**Comment by Aldaviva** [comment]
Thank you for alerting me to this change. I have personally migrated from Cygwin sshd to the Windows distribution, so it's very helpful for you to raise this because I wouldn't have noticed.

I will update the default configuration file with this change.

**Comment by Aldaviva** [comment]
Released in version [1.3.1](https://github.com/Aldaviva/Fail2Ban4Win/releases/tag/1.3.1) along with migration instructions.

It turned out that the payload format also changed for the Microsoft distribution of OpenSSH sshd, so I updated that configuration too.
